mono/mini/mini-amd64-gsharedvt.c

```diff
@@ -148,11 +149,13 @@ get_arg_slots (ArgInfo *ainfo, int **out_slots, gboolean is_source_argument)
 		src [0] = map_reg (sreg);
 		break;
 	case ArgValuetypeInReg:
+	case ArgValuetypeAddrInIReg:
 		nsrc = ainfo->nregs;
```

Comment:
In ArgValuetypeAddrInIReg ainfo->nregs will always be 1 so I reused the case to reduce code duplication. I can split into a separate case with nsrc set to 1 if that makes the intent clearer, but with some code duplication.